doc/user-guides/auth/anonymous-access.md

```diff
@@ -0,0 +1,159 @@
+# Enforcing anonymous access with Kuadrant AuthPolicy
+
+Learn how to allow anonymous access to certain endpoints using Kuadrant's `AuthPolicy`
+
+## Prerequisites
+
+You have installed Kuadrant in a [kubernetes](https://docs.kuadrant.io/latest/kuadrant-operator/doc/install/install-kubernetes/) or [OpenShift](https://docs.kuadrant.io/latest/kuadrant-operator/doc/install/install-openshift/) cluster.
+
+## Run the guide ① → ⑦
+### ① Deploy Toy Store application
+
+Deploy a simple HTTP application service that echoes back the request data:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/Kuadrant/kuadrant-operator/refs/heads/main/examples/toystore/toystore.yaml
+```
+
+### ② Expose the Application
+
+Create an `HTTPRoute` to expose an `/cars` and `/public` path to the application:
+
+```sh
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: toystore
+spec:
+  parentRefs:
+  - name: kuadrant-ingressgateway
```

Comment:
i believe the gateway doesnt get created with the install k8s and install openshift docs the make target is the only one that would create it or the quickstart

Reply:
Oh, good catch, I'll add the creation of the Gateway as part of the guide 👍